pkg/model/application.proto

```diff
@@ -43,6 +43,8 @@ message Application {
     // This must be one of the provider names registered in the piped.
     // TODO: Add validation for this field.
     string platform_provider = 15;
+
+    repeated string deploy_targets = 16;
```

Comment:
How about adding comments and a validation?
The validation may be like `min_len = 1`

Reply:
This should not have validation, or application which created by pipedv0 will be invalided :eyes:

Reply:
Oops, that's right 🙏🏻

Reply:
Also, could you add comment to explain what does this field do or be used for @ffjlabo

Reply:
Forgot to  add comment 🙏  will do it.

Reply:
@Warashi  @khanhtc1202 fied f31a246